src/Opdex.Platform.WebApi/Controllers/DeployController.cs

```diff
@@ -52,7 +51,6 @@ public DeployController(IMediator mediator, ILogger<DeployController> logger, Op
         /// <param name="cancellationToken">cancellation token.</param>
         /// <returns>Success</returns>
         [HttpPost("dev-contracts")]
-        [Authorize(Policy = "AdminOnly")]
```

Comment:
This needs to come back

Reply:
Cool, didn't realise the market address wasn't required to auth with an admin key. Re-added it.